library/psa_crypto.c

```diff
@@ -4631,11 +4631,7 @@ psa_status_t psa_cipher_decrypt(mbedtls_svc_key_id_t key,
         goto exit;
     }
 
-    if (alg == PSA_ALG_CCM_STAR_NO_TAG &&
-        input_length < PSA_BLOCK_CIPHER_BLOCK_LENGTH(slot->attr.type)) {
```

Comment:
If the issue is with the fact that the input_length (lengths of IV (default 13) + message?) is less than the block length (which is 16) IF the message is < 3 bytes, I don't understand why the fix doesn't lie within `PSA_BLOCK_CIPHER_BLOCK_LENGTH`

If it is okay to remove `...input_length < PSA_BLOCK_CIPHER_BLOCK_LENGTH(slot->attr.type)) {`, why was it there in the first place?